sale_product_pack/__manifest__.py

```diff
@@ -14,7 +14,7 @@
     'license': 'AGPL-3',
     'depends': [
         'product_pack',
-        'sale',
+        "sale_management",
```

Comment:
Hi @ernestotejeda, @ivantodorovich do you have any suggestion about module dependency, as mentioned in this PR description I could move this fix in a new module `sale_management_product_pack` auto installable ?

(@ivantodorovich the moka's task #1850)